### PR TITLE
Make unrevised content more visible for logged in users

### PR DESCRIPTION
**Pull request by elbotho**
as requested by @yolo-bird this PR shows unrevised content in the taxonomy and also renders a note when viewing the content directly:

image
image

### status

this is step one for `articles`, `videos`, `applets`, `courses` only.

`exercises`, `solutions`, `grouped…`, `coursePages` need more work and concept.

### CODE REVIEW

**Comment by ghost** [comment]
Thanx for the draft @elbotho ! 

Some suggestions: 

- [x] I think we could use another icon so that users see immediately that this content is work in progress. Maybe a construction work sign

I would already show the unrevised content in preview for new content with a suitable information box on top
- [ ] This content is new and has not been reviewed yet. Please be aware that it may be incomplete, contain mistakes or may not be conform with our community guidelines

For existing content I see two possible ways to do it. 
1) Show the unrevised content with the following information box on top:
- [ ] This revision is new and has not been reviewed yet. Please be aware that it may be incomplete, contain mistakes or may not be conform with our community guidelines. To view the latest reviewed revision click here. 
2) Show the current revised version of the content with the information box:
- [ ] There is a newer revision that has not been reviewed yet. To view it please click here. 

For exercises and exercise groups inside a folder we for sure need some visual clue so that it is clear that only for this exercise or even for a subtask of an exercise there is a new unrevised revision. 

What do you think?

**Comment by elbotho** [comment]
@yolo-bird thank you for your feedback, I replied in slack.
This PR now targets articles, videos, applets, courses only (see first comment).

**Review by Entkenntnis — APPROVED** [review]
@elbotho Thank you for working on this. Tested it locally and it worked quite nicely.

Feel free to merge if suitable.

File: src/components/taxonomy/sub-topic.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+export function SubTopic({ data, subid, id }: SubTopicProps) {
+  return (
+    <>
+      <h2 className="my-3 serlo-h2 border-0" style={{ hyphens: 'auto' }}>
+        <Link href={data.url} path={[subid, 'title']}>
+          {data.title}
+        </Link>
+      </h2>
+
+      <Wrapper className="mb-5 pb-6 flex flex-col sm:flex-row border-b-1 border-gray-300">
```
Comment (Entkenntnis) [inline src/components/taxonomy/sub-topic.tsx]: small note: `border-b` is the same as `border-b-1` (not adding a number is defaulting to 1px, so we can do this without changing the theme.)

**Comment by elbotho** [comment]
We decided to go with this for now and wait for feedback before working on courses and exercises.
Also displaying the users unrevised revision in their profile for example might be a better approach anyway.

**Pull request merged**